src/order_gui.cpp

```diff
@@ -1465,6 +1468,41 @@ struct OrdersWindow : public Window {
 		return true;
 	}
 
+	/**
+	 * Clones an order list from a vehicle list.
```

Comment:
This could also use a description of when this fires, as above.

Reply:
Done.